Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -58,96 +58,33 @@ extension Store {
   }
 }
 
-extension ViewStore {
-  @available(
-    *, deprecated,
-    message:
-      "Dynamic member lookup is no longer supported for bindable state. Instead of dot-chaining on the view store, e.g. 'viewStore.$value', invoke the 'binding' method on view store with a key path to the value, e.g. 'viewStore.binding(\\.$value)'. For more on this change, see: https://github.com/pointfreeco/swift-composable-architecture/pull/810"
-  )
-  public subscript<Value>(
-    dynamicMember keyPath: WritableKeyPath<State, BindableState<Value>>
-  ) -> Binding<Value>
-  where Action: BindableAction, Action.State == State, Value: Equatable {
-    self.binding(
-      get: { $0[keyPath: keyPath].wrappedValue },
-      send: { .binding(.set(keyPath, $0)) }
-    )
-  }
-}
-
-// NB: Deprecated after 0.25.0:
-
 #if compiler(>=5.4)
-  extension BindingAction {
```

Comment:
We want to keep these deprecations around for Xcode 12.5+, they are old interfaces.

Reply:
This is because also github's diff visualizer 😢 . It's in the same line (one below actually) within the file.

Reply:
The diff is 16 additions and 79 deletions. Even if the visualizer was tripping up, the additions should be 2 more than whatever deletions are detected. It looks like maybe your changes weren't pushed?

Reply:
The # of deletions are higher, because I removed the deprecations, in `#else` part. Because they were using `BindingAction` and `BindableState` which is not defined for compiler versions < 5.4. However diff calculation got weird because of the similar texts. Although, I did not changed anything in this `line 81`, it seems to be removed and re-added in `line 82` in the new file.